controller/bookmark.js

```diff
@@ -4,7 +4,10 @@ const uuidv1 = require("uuid/v1")
 module.exports = {
   getBookmarks: (req, res, next) => {
     res.locals.response = Object.assign({}, res.locals.response || {}, {
-      bookmark: db.get("bookmarks").value()
+      bookmark: db
+        .get("bookmarks")
+        .sortBy({ default: "createdAt", url: "-url" })
```

Comment:
Take a look at issue #4 `createdAt` is just the default value. We also want to be able to sort by `url`